src/training/data.py

```diff
@@ -27,6 +27,54 @@
     hvd = None
 
 
+class TextPairDataset(Dataset):
+    def __init__(self, input_filename, text_a_key, text_b_key, tokenizer=None):
+        logging.debug(f'Loading parquet data from {input_filename}.')
+        df = pd.read_parquet(input_filename)
```

Comment:
this is unlikely to scale

Reply:
Right. I will update this. Do you have any suggestions?